src/System.Reflection.Metadata/specs/PE-COFF.md

```diff
@@ -65,3 +65,41 @@ UncompressedSize shall be greater than 0. Other values are reserved for future u
 The Major version specified in the entry indicates the version of the Portable PDB format. The Minor version indicates the version of the Embedded Portable PDB data format.
 
 The value of Stamp field in the entry shall be 0.
+
+### PDB Checksum Debug Directory Entry (type 18)
+
+Stores crypto hash of the content of the symbol file the PE/COFF file was built with.
+
+The hash can be used to validate that a given PDB file was built with the PE/COFF file and not altered in any way. 
+
+More than one entry can be present, in case multiple PDBs were produced during the build of the PE/COFF file (e.g. private and public symbols). 
```

Comment:
I am assuming the 'Signature' field will also be duplicated. Is that correct?

Reply:
Yeah, there would be multiple entries (rows) in the debug directory table: 


Offset | Size | Field | Description
-- | -- | -- | --
0 | 4 | Characteristics | Reserved, must be zero.
4 | 4 | TimeDateStamp | The time and date that the debug data was created.
8 | 2 | MajorVersion | The major version number of the debug data format.
10 | 2 | MinorVersion | The minor version number of the debug data format.
12 | 4 | Type | The format of debugging information. This field enables   support of multiple debuggers. For more information, see section 6.1.2,   “Debug Type.”
16 | 4 | SizeOfData | The size of the debug data (not including the debug   directory itself).
20 | 4 | AddressOfRawData | The address of the debug data when loaded, relative to the   image base.
24 | 4 | PointerToRawData | The file pointer to the debug data.


The data above are pointed to by `AddressOfRawData`/`PointerToRawData`